about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" 
+  integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" 
+  crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300&display=swap"
+    rel="stylesheet"
+  />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Secular+One&display=swap"
+    rel="stylesheet"
+  />
+  <link rel="stylesheet" href="style.css">
+  <title>TechnoCamp_UK_Summit_2022</title>
+</head>
+<body>
+  <header class="page-header">
+    <div class="mobile-menu">
+      <button type="button" aria-label="open menu" class="open-menu open">
+        <i class="fa fa-bars"></i>
+      </button>
+      <nav class="mobile-nav">
+        <button type="button" aria-label="close menu closebtn" class="close-menu close">
+          <i class="fas fa-times"></i>
+        </button>
+        <ul>
+          <li><a class="close active" href="./index.html">Home</a></li>
+          <li><a class="close" href="./about.html">About</a></li>
+          <li><a class="close" href="#">Service</a></li>
+          <li><a class="close" href="#">Blogs</a></li>
+          <li><a class="close" href="#">Support</a></li>
+          <li><a class="close" href="#">News</a></li>
+          <li class="bordered-link"><a class="close" href="#">CONTACT US</a></li>
+        </ul>
+      </nav>
+       <img class="mobile-logo" src="./Images/tech-camp-logo.png" alt="Techno_Logo"> 
+    </div>
+    <div class="social-icons">
+      <i class="fab fa-facebook-f"></i>
+      <i class="fab fa-twitter"></i>
+      <a class="lang" href="about.html">My Page</a>
+    </div>
+    <div class="main-header">
+      <div class="desktop-logo">
+        <img src="./Images/tech-camp-logo.png" alt="Techno_Logo">
+      </div>      
+      <ul class="main-header-content">
+        <li><a class="close active" href="./index.html">Home</a></li>
+        <li><a class="close" href="./about.html">About</a></li>
+        <li><a class="close" href="#">Service</a></li>
+        <li><a class="close" href="#">Blogs</a></li>
+        <li><a class="close" href="#">Support</a></li>
+        <li><a class="close" href="#">News</a></li>
+        <li class="bordered-link"><a class="close" href="#">CONTACT US</a></li>
+      </ul>
+    </div>
+  </header>
```

Comment:
- [ ] On the navbar on mobile view I suggest you remove the logo because it is not really useful if you keep it there and also you can still push the hamburger menu to the left because it is more at the center than to the left.